src/main/resources/templates/home.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Microblogger</title>
+</head>
+<body>
+{{#currentUser}}
+<form action="/logout" method="post" style="display: inline">
+    <button type="submit">Logout</button>
+</form>{{currentUser}} is currently logged in
+<hr>
+<form action="/add-message" method="post">
+    <input type="text" placeholder="Enter your message" name="content" />
+    <button type="submit">Submit</button>
+</form>
+<h1>Messages by all users:</h1>
+<form id="fDelete" action="/delete-message" method="post">
```

Comment:
This should be removed since delete has been moved to the detail pages.  Be careful simply changing code without making sure to clean up after yourself in the change.